Makefile

```diff
@@ -101,17 +129,23 @@ install-proto-bin: install-vendor-dep
 	go install $(package_root)/$(vendor_prefix)/$(protoc_go_package)
 
 .PHONY: mock-gen
-mock-gen: install-mockgen install-license-bin install-util-mockclean
-	@echo "+ $@, generating mocks"
-	PACKAGE=$(package_root) $(auto_gen) $(mocks_output_dir) $(mocks_rules_dir)
+mock-gen: install-ci-tools mock-gen-no-deps
+	@echo "+ $@"
 
-.PHONY: mock-gen-deps
+.PHONY: mock-gen-no-deps
 mock-gen-no-deps:
 	@echo "+ $@"
-	PACKAGE=$(package_root) $(auto_gen) $(mocks_output_dir) $(mocks_rules_dir)
+	@echo generating mocks
+	PATH=$(retool_bin_path):$(PATH) PACKAGE=$(package_root) $(auto_gen) $(mocks_output_dir) $(mocks_rules_dir)
 
 .PHONY: all-gen
-all-gen: mock-gen code-gen
+all-gen: mock-gen kubernetes-gen
+
+# Ensure base commit had up-to-date generated artifacts
+.PHONY: verify-all-gen
+verify-all-gen: mock-gen kubernetes-gen
```

Comment:
nit: mind sticking to calling it `test-all-gen` (just to stay uniform with other repos)

Reply:
Will do